content/packages-and-modules/securing-your-code/verifying-the-pgp-signature-for-a-package-from-the-npm-public-registry.mdx

```diff
@@ -28,13 +28,13 @@ To ensure the integrity of a package version you download from the npm public re
 1. On the command line, fetch the signature for the package version you want and save it in a file:
 
    ```
-   $ http GET https://registry.npmjs.org/light-cycle | json "versions['1.4.3'].dist.npm-signature" > sig-to-check
+   npm view light-cycle@1.4.3 --json | npm exec json ".dist.npm-signature" > sig-to-check
    ```
 
 2. Get the integrity field for that version (example below includes response):
 
    ```
-   $ http GET https://registry.npmjs.org/light-cycle | json "versions['1.4.3'].dist.integrity"
+   npm view light-cycle@1.4.3 --json | npm exec json ".dist.integrity"
```

Comment:
```suggestion
   npm view light-cycle@1.4.3 dist.integrity
```

Reply:
Not sure why I thought that would return the quoted value, TIL.